codegenerator/cli/templates/static/codegen/src/bindings/Fuel.res

```diff
@@ -0,0 +1,30 @@
+@genType.import(("./OpaqueTypes.ts", "EthersAddress"))
+type fuelAddress = Ethers.ethAddress
```

Comment:
Is it the same though? Still fixed 20 bytes  that can be represented as checksummed hexadecimal? Ie if I run it through viem.getAddress it's not going to runtime error?

Reply:
It'll be an error. But this is how we typed it in the fork. Should be fixed later